### PR TITLE
Query unhealthy GPU from prometheus

### PR DESCRIPTION
**Pull request by mzmssg**
A part of #2192

After #2209, unhealthy gpu info will be collected by prometheus. This PR provides following operation guide after alerting.

In our design, admin need to do:
1. Query prometheus, collect unhealthy nodes.
2. Add unhealthy nodes to blacklist.
3. Enforce service to reload the blacklist.

More details in https://github.com/Microsoft/pai/blob/8b186958f9ed55c4c0fac157977f58f9b56adc8c/docs/tools/node_maintain.md

### CODE REVIEW

File: docs/tools/node_maintain.md [hunk 1]
```diff
@@ -2,45 +2,60 @@
...
 
-Graceful decommissioning could minimize the impact on existing job when removing unhealthy nodes. 
-This is a tool to help you achieve it in OpenPAI. 
+This [tool](../../src/tools/node_maintain.py) help you to gracefully decommission unhealthy nodes in OpenPAI. 
 
-In OpenPAI, an unhealthy node-list records the nodes to be decommissioned. 
-Components could sync the list and decommission them as needed.
-
-**NOTES: For now, only YARN nodes are decommissioned, HDFS doesn't.**
+**NOTES: This tool should be invoked in the dev-box, under `{PAI_ROOT_DIR}/src/tools`. For now, it only supports decommissioning YARN nodes.**
```
Comment (hao1939) [inline docs/tools/node_maintain.md]: So HDFS node was not decommissioned, suggest to point it out.

File: docs/tools/node_maintain.md [hunk 2]
```diff
@@ -2,45 +2,60 @@
...
+python node_maintain.py {object} {action} {arguments}
+```
+
+### Get current unhealthy GPU nodes
 
-So, a common invoking looks like:
+###### 1. Query prometheus by this tool
 ```bash
-python node_maintain.py {master_ip} sub-command sub-command-arguments
+python node_maintain.py badgpus get -m {prometheus_ip} [--prometheus-port {prometheus-port}]
```
Comment (hao1939) [inline docs/tools/node_maintain.md]: We'd better add some background about the why we 'query prometheus'. For example: We monitor some gpu metrics using prometheus, we can query the bad GPUs from prometheus.

File: docs/tools/node_maintain.md (hunk 1 above)
Comment (mzmssg) [reply docs/tools/node_maintain.md]: Added

File: docs/tools/node_maintain.md (hunk 2 above)
Comment (mzmssg) [reply docs/tools/node_maintain.md]: Added

**Pull request merged**